### PR TITLE
BRMS-300: handling 413 and Forbidden response codes

### PR DESCRIPTION
**Pull request by adamconder**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,47 +25,135 @@ Accept       | `String` | application/vnd.hmrc.1.0+json   | N/A  | API Version
...
+HTTP/1.1 400 Bad Request
+Accept: application/vnd.hmrc.1.0+json
+Cache-Control: no-cache,no-store,max-age=0
+Content-Type: application/json; charset=utf-8
 
-```json
 {
-  "matched": true
+  "code":"INVALID_BIRTH_REFERENCE_NUMBER",
+  "reason":"The birth reference number does not meet the required length"
```
Comment (adamconder) [inline README.md]: @chris-ianson should this be code and message? https://developer.service.hmrc.gov.uk/api-documentation/docs/reference-guide#errors

File: app/uk/gov/hmrc/brm/utils/BrmException.scala [hunk 2]
```diff
@@ -48,10 +48,16 @@ trait BRMException extends Controller {
 
   def forbiddenPF(message: String)(implicit payload: Payload) : PartialFunction[Throwable, Result] = {
     case Upstream4xxResponse(m, FORBIDDEN, _, _) if m.contains("INVALID_DISTRICT_NUMBER") =>
-      logException(Some(message), Some(s"Forbidden returned from NRS message"), BAD_REQUEST)
+      logException(Some(message), Some(m), BAD_REQUEST)
       BadRequest
```
Comment (adamconder) [inline app/uk/gov/hmrc/brm/utils/BrmException.scala]: @manishsw  should this be a 200 response code?

File: README.md (hunk 1 above)
Comment (chris-ianson) [reply README.md]: Yes, this would align with standards. Do you want us to change it?

File: app/uk/gov/hmrc/brm/utils/BrmException.scala (hunk 2 above)
Comment (manishsw) [reply app/uk/gov/hmrc/brm/utils/BrmException.scala]: @adamconder Ya we can send 200 false. Also if we get information what happens for bad request in dfs side.

File: app/uk/gov/hmrc/brm/utils/ErrorResponses.scala [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  * Created by mew on 07/04/2017.
+  */
+object ErrorResponses {
+
+  case class ErrorResponse(code: String, message: String) {
+    def json = Json.parse(
+      s"""
+         |{
+         |  "code": "$code",
+         |  "messagei": "$message"
```
Comment (manishsw) [inline app/uk/gov/hmrc/brm/utils/ErrorResponses.scala]: @adamconder  is this correct messagei?

**Pull request merged**